x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/helpers.ts

```diff
@@ -202,23 +205,22 @@ export const getEnrichedFieldMetadata = ({
     return {
       indexFieldName: field,
       indexFieldType: type,
-      indexInvalidValues,
+      indexInvalidValues: [],
```

Comment:
For reviewers: 

Just making it a little more explicit that this branch of logic only has 2 dynamic values `field` and `type` the rest is hard coded and constitutes a separate `CustomFieldMetadata` type with a discriminant in `hasEcsMetadata` field